app/admin/products.rb

```diff
@@ -5,7 +5,8 @@
   actions :all, except: [:destroy]
   permit_params :id, :user_email, :create_another
 
-  index do
+  # To do: set up index file export
+  index download_links: false do
```

Comment:
```suggestion
  index do
```

Reply:
Let's keep the CSV export in place and disable fields later. We can use this as a way to quickly QA the bulk import workflow.